ledger/src/blockstore.rs

```diff
@@ -6690,6 +6694,7 @@ pub mod tests {
             rewards: Some(rewards_vec.clone()),
             loaded_addresses: test_loaded_addresses.clone(),
             return_data: Some(test_return_data.clone()),
+            compute_units_consumed: Some(3812649u64),
```

Comment:
Please use a variable for this value so that it can be reused later in the assertion